test/integration/integration-tests.js

```diff
@@ -207,6 +208,16 @@ test('Gets organization for tests which change and delete data', (t) => {
     })
 })
 
+test.only('Gets organization for tests which change and delete data', (t) => {
+  return client.getOrganizations()
+    .then((response) => {
+      const organization = response.items[0]
+      return Promise.all([
+        organizationSpaceMembershipTests(t, organization)
```

Comment:
I do not think we need to get the organization two times, you can probably combine the organizationSpaceMembershipTests with the organizationMembershipTests. So move this line in the Promise.all([organizationMembershipTests(t, org), organizationSpaceMembershipTests(t, org)])

Reply:
@vbeleuta oh! yeah! totally. thx a lot for your feedback. 🙂

Reply:
Done 🙂